sanssouci/post_hoc_bounds.py

```diff
@@ -38,33 +38,34 @@ def max_fp(p_values, thr):
         Annals of Statistics, 48(3), 1281-1303.
     """
 
-    # make sure that thr is sorted
-    if np.linalg.norm(thr - thr[np.argsort(thr)]) > 0.0001:
-        thr = thr[np.argsort(thr)]
-        print("The input thr was not sorted -> this is done now")
+    # make sure that thresholds is sorted
+    if np.linalg.norm(thresholds - thresholds[np.argsort(thresholds)]) \
+       > 0.0001:
+        thresholds = thresholds[np.argsort(thresholds)]
+        print("The input 'thresholds' was not sorted -> this is done now")
 
     # do the job
-    nS = p_values.shape[0]
-    K = thr.shape[0]
-    size = np.min([nS, K])
+    subset_size = p_values.shape[0]
+    K = thresholds.shape[0]
```

Comment:
Can you call that `n_tests` (?) instead of `K` ?

Reply:
`K` is not the number of tests but the size of the template, so if we want an explicit name I'd suggest `template_size` or something alike.